lab_1/main.py

```diff
@@ -40,7 +69,18 @@ def get_top_n_words(freq_dict: dict, top_n: int) -> list or None:
     :param top_n: a number of the most common words
     :return: a list of the most common words
     """
-    pass
+
+    if not isinstance(freq_dict, dict) or not isinstance(top_n, int):
+        return None
+    top_n_words = []
+    freq_dict_keys = [key for key in freq_dict]
+    for i in freq_dict_keys:
+        top_n_words = sorted(freq_dict, key=freq_dict.get, reverse=True)[:top_n]
```

Comment:
good, but you can sort keys just here without the previous lines
do you really need a loop?

Reply:
explain what happens here, please